lms/static/sass/_header.scss

```diff
@@ -0,0 +1,344 @@
+/*
+  This file contains all the necessary styling for a uniform
+  navigation bar that can exist on any of the v1, v2 or bootstrap
+  pages.
+*/
+.global-header {
+  box-sizing: border-box;
+  width: 100%;
+  overflow: hidden;
+  padding-bottom: $baseline/2;
+  border-bottom: 1px solid theme-color("primary");
+  box-shadow: 0 1px 5px 0 rgba(0, 0, 0, 0.1);
+  background: theme-color("inverse");
+
+  /*
+    Logo and course identification block
+  */
+  .header-logo {
+    display: inline;
+
+    a {
+      @include float(left);
+      @include margin($baseline/2, 0, 0, $baseline);
+
+      display: block;
+
+      .logo {
+        @include float(left);
+
+        height: $header-logo-size;
+      }
+    }
+
+    .course-header {
+      @include float(left);
+      @include margin($baseline*1.25, 0, 0, $baseline);
+
+      font-size: $font-size-lg;
+      color: theme-color("dark");
+      line-height: 1em;
+
+      .provider {
+        font-weight: $font-weight-bold;
+      }
+
+      // Hide the course header for smaller screen sizes
+      @include media-breakpoint-down(md) {
```

Comment:
I'd rather hide by default and then have the header appear for larger sizes.

Reply:
done